src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private int sideA;
+    private int sideB;
+
+    RightTriangle(String color, int sideA, int sideB) {
+        super(color);
+        this.sideA = sideA;
+        this.sideB = sideB;
+    }
+
+    @Override
+    public void returnInfo() {
+        System.out.println("Figure: RightTriangle, area: " + sideA * sideB / 2
```

Comment:
All magic numbers in your code should be constants. The number '2' used in the area calculation should be declared as a constant. For example, you can create a constant named `HALF` with a value of 2.